### ISSUE TITLE
Completely unrelated...

### ISSUE THREAD

**Issue by BourgeoisBear** [taylodl/listp#1]
Was this your comment?

https://news.ycombinator.com/item?id=35787371

If so, absolutely brilliant!

**Comment by taylodl** [comment]
Yes, that was me. Just happened to have the right experience and thought at the time! I had just gone though an experience with LinkedIn where they needed to verify my identity by providing a photo of the front and back of my drivers license. That's when I realized as an identity provider they can now make a claim on an access token with regards to my age and they have the evidence to substantiate that claim should it ever be challenged. That way I do this verification with LinkedIn (or Google, or whomever) and they can make that claim to any website requiring age verification. All I have to do is log in with LinkedIn or some other identity provider of my choosing. When you think about it, it makes sense that an identity provider would be able to verify your age since that's an aspect of your identity.
